tests/upgrade_k8s_test.go

```diff
@@ -120,6 +121,14 @@ var _ = Describe("k3s upgrade test", Label("upgrade-k8s"), func() {
 			return out
 		}, 900*time.Second, 10*time.Second).Should(ContainSubstring("system-upgrade-controller"))
 
+		By("wait for all containers to be in running state")
+		Eventually(func() string {
+			out, _ := kubectl(vm, "get pods -A")
+			fmt.Printf("out = %+v\n", out)
```

Comment:
For the future (since this is now merged), this will print the output for every invocation of this `Eventually`. It's better to define the `out` variable outside this block, set its value in this block and only print it when `Eventually` fails (as a second argument to `ShouldNot`).

Reply:
Sorry this is a leftover of my testing, wanted to see on each invocation when exactly it finally printed as I was having issues getting those pods to appear (slowness) :/